src/ClientWrapper.php

```diff
@@ -25,7 +25,7 @@ public function __construct($baseUrl, $logger=null)
      * @return array|mixed
      * @throws LauraException
      */
-    public function request($urlArray, $header, $data)
+    public function request($urlArray, $header, $data, $jsonEncode = true)
```

Comment:
Why do we need this flag?

Reply:
curl --location --request POST 'https://open-api.guesty.com/oauth2/token' \
--header 'accept: application/json' \
--data-urlencode 'grant_type=client_credentials' \
--data-urlencode 'scope=open-api' \
--data-urlencode 'client_secret={application client SECRET}'
--data-urlencode 'client_id={application client ID}' \

Reply:
others request using

curl --request POST \
     --url https://open-api.guesty.com/v1/listings \
     --header 'Accept: application/json' \
     --header 'Content-Type: application/json' \
     --data '
{
     "title": "dddd",
     "origin": "bbb"
}
'